src/__tests__/cdfDatasource.test.ts

```diff
@@ -122,18 +117,19 @@ describe('CDF datasource', () => {
 
   describe('reduce timeseries', () => {
     it('should return datapoints and the default label', () => {
+      const id = 2;
       const metaResponses: any[] = [
         {
           result: getDataqueryResponse({
-            items: [{ externalId: 'Timeseries123' }],
+            items: [{ id }],
             aggregates: ['average'],
           }),
-          metadata: getMeta('Timeseries123', 'average', ['']),
+          metadata: getMeta(id, 'average', ['']),
         },
       ];
       const [reduced] = reduceTimeseries(metaResponses, [1549336675000, 1549337275000]);
       expect(reduced.datapoints).toEqual([[0, 1549336675000], [1, 1549337275000]]);
-      expect(reduced.target).toEqual('average Timeseries123');
+      expect(reduced.target).toEqual(`average ${id}`);
```

Comment:
try to use external id, so we have more complex real test case (other places as well)